Spanner/src/Operation.php

```diff
@@ -203,6 +203,12 @@ public function execute(Session $session, $sql, array $options = [])
         $options += $this->mapper->formatParamsForExecuteSql($parameters, $types);
 
         $context = $this->pluck('transactionContext', $options);
+        if (!empty($options['tags']['transactionTag'])) {
+            $context = [
```

Comment:
I don't understand why `tag` is being added to the transaction context..

FWICT, it seems to just be for testing purposes which suggests that this shouldn't be done.